.github/workflows/ci.yml

```diff
@@ -162,6 +162,19 @@ jobs:
         DefaultDeviceTimeoutSec=180
         EOF
 
+        tee mkosi.configure <<EOF
+        echo "Hello from inside mkosi.configure!" >&2
+        cat
+        EOF
+
+        for script in sync prepare build postinst finalize postoutput clean
```

Comment:
This overrides the existing `mkosi.postinst` script no?

Reply:
And the prepare, too, dropped both from the list.